lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -230,6 +232,32 @@ def truncate_message(message)
 
         message.byteslice(0...MAX_BYTES)
       end
+
+      def minimum_log_level
+        if Logger::Severity.constants.include?(configured_log_level_constant)
+          configured_log_level_constant
+        else
+          :DEBUG
+        end
+      end
+
+      def configured_log_level_constant
+        format_log_level_constant(NewRelic::Agent.config[LOG_LEVEL_KEY])
+      end
+
+      def format_log_level_constant(log_level)
+        log_level.upcase.to_sym
+      end
+
+      def severity_too_low?(severity)
+        severity_constant = format_log_level_constant(severity)
+        # always record custom log levels
+        return false unless Logger::Severity.constants.include?(severity_constant)
+
+        Logger::Severity.const_get(severity_constant) < Logger::Severity.const_get(minimum_log_level)
+      rescue NameError => e
```

Comment:
Can we remove this `rescue` now that both methods that perform a constant lookup use a `constants.include?` guard? If not, we should have a test that explains a pathway to reaching the `rescue`.

Reply:
Great point! Fixed in 93eae88624a860ac88fe6420dff8e4aa6c5a1f1d